packages/grafana-ui/src/saga-themes/tokens/core.json

```diff
@@ -0,0 +1,893 @@
+      },
+      "900": {
+        "value": "#162410",
+        "type": "color"
+      }
+    },
+    "yellow": {
+      "0": {
+        "value": "#f5f5eb",
+        "type": "color"
+      },
+      "100": {
+        "value": "#f8f7b0",
+        "type": "color"
+      },
+      "200": {
+        "value": "#fcf574",
+        "type": "color"
+      },
+      "300": {
+        "value": "#ffec3d",
+        "type": "color"
+      },
+      "400": {
+        "value": "#d9dd25",
+        "type": "color"
+      },
+      "500": {
+        "value": "#a6ba16",
+        "type": "color"
+      },
+      "600": {
+        "value": "#7b980f",
+        "type": "color"
+      },
+      "700": {
+        "value": "#56750b",
+        "type": "color"
+      },
+      "800": {
+        "value": "#385309",
+        "type": "color"
+      },
+      "900": {
+        "value": "#1e3006",
+        "type": "color"
+      }
+    },
+    "orange": {
+      "0": {
+        "value": "#fffbf7",
+        "type": "color"
+      },
+      "100": {
+        "value": "#ffdfc0",
+        "type": "color"
+      },
+      "200": {
+        "value": "#ffc58a",
+        "type": "color"
+      },
+      "300": {
+        "value": "#ffad5a",
+        "type": "color"
+      },
+      "400": {
+        "value": "#ff9830",
+        "type": "color"
+      },
+      "500": {
+        "value": "#d87d21",
+        "type": "color"
+      },
+      "600": {
+        "value": "#b16619",
+        "type": "color"
+      },
+      "700": {
+        "value": "#8b5015",
+        "type": "color"
+      },
+      "800": {
+        "value": "#643a11",
+        "type": "color"
+      },
+      "900": {
+        "value": "#3d250c",
+        "type": "color"
+      }
+    },
+    "gray": {
+      "0": {
+        "value": "#ffffff",
+        "type": "color"
+      },
+      "100": {
+        "value": "#e9ebed",
+        "type": "color"
+      },
+      "200": {
+        "value": "#d4d7dc",
+        "type": "color"
+      },
+      "300": {
+        "value": "#bfc4ca",
+        "type": "color"
+      },
+      "400": {
+        "value": "#abb1b8",
+        "type": "color"
+      },
+      "500": {
+        "value": "#989ea7",
+        "type": "color"
+      },
+      "600": {
+        "value": "#858c95",
+        "type": "color"
+      },
+      "700": {
+        "value": "#737a83",
+        "type": "color"
+      },
+      "800": {
+        "value": "#626872",
+        "type": "color"
+      },
+      "900": {
+        "value": "#515660",
+        "type": "color"
+      },
+      "1000": {
+        "value": "#41464e",
+        "type": "color"
+      },
+      "1100": {
+        "value": "#31353d",
+        "type": "color"
+      },
+      "1200": {
+        "value": "#22252b",
+        "type": "color"
+      },
+      "1300": {
+        "value": "#131415",
+        "type": "color"
+      },
+      "1400": {
+        "value": "#000000",
+        "type": "color"
+      }
+    }
+  },
+  "boder": {
```

Comment:
Should read `border`

Reply:
That was corrected! thanks for the detail! ;)